packages/bn128/bn128.0.1.3/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1.2"
+maintainer: "Dwight Guth <dwight.guth@runtimeverification.com>"
+authors: "Dwight Guth <dwight.guth@runtimeverification.com>"
+homepage: "https://github.com/runtimeverification/bn128-ml"
+bug-reports: "https://github.com/runtimeverification/bn128-ml/issues"
+license: "UIUC"
+dev-repo: "https://github.com/runtimeverification/bn128-ml.git"
+build: [make]
+install: [make "install"]
+remove: ["ocamlfind" "remove" "bn128"]
+depends: ["ocamlfind" "zarith"]
```

Comment:
your packages doesn't seems to use the Findlib library. Could you please flag the `ocamlfind` dependency with the `{build}` attribute?